packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -229,11 +241,14 @@ class Authenticator extends StatefulWidget {
     this.onException,
     this.exceptionBannerLocation = ExceptionBannerLocation.auto,
     this.preferPrivateSession = false,
+    this.initialStep = AuthenticatorStep.signIn,
+    AuthenticatorBuilder? builder,
```

Comment:
Something like this could also work:

```dart
@override
Widget build(BuildContext context) {
  return Authenticator(
    child: MaterialApp(
      builder: Authenticator.builder((context, state, defaultScreen) {
        switch (state) {
          case AuthenticatorStep.loading:
            return const CustomLoadingScreen();
        }
        return defaultScreen;
      }),
      home: const RouteA(),
    ),
  );
}
```